public/stylesheets/carousel.css

```diff
@@ -0,0 +1,123 @@
+
+
+* {
+	box-sizing: border-box;
+}
+
+a:link,
+a:hover,
+a:active,
+a:visited {
+	transition: color 150ms;
+	color: #95a5a6;
+	text-decoration: none;
+}
+
+a:hover {
+	color: #7f8c8d;
+	text-decoration: underline;
+}
+
+.contain {
+	width: 100%;
+}
+
+.row {
+	overflow: scroll;
+	width: 100%;
+}
+
+.row__inner {
+	transition: 450ms transform;
+	font-size: 0;
+	white-space: nowrap;
+	margin: 70.3125px 0;
+	padding-bottom: 10px;
+}
+
+.tile {
+	position: relative;
+	display: inline-block;
+	width: 250px;
+	height: 140.625px;
+	margin-right: 10px;
+	font-size: 20px;
+	cursor: pointer;
+	transition: 450ms all;
+	transform-origin: center left;
+}
+
+.tile__img {
+	width: 250px;
+	height: 140.625px;
+	object-fit: cover;
+}
+
+.tile__details {
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	top: 0;
+	font-size: 10px;
+	opacity: 0;
+	background: linear-gradient(to top, rgba(0,0,0,0.9) 0%, rgba(0,0,0,0) 100%);
+	transition: 450ms opacity;
+}
+
+.tile__details:after,
+.tile__details:before {
+	content: '';
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	display: #000;
+}
+
+.tile__details:after {
+	margin-top: -25px;
+	margin-left: -25px;
+	width: 50px;
+	height: 50px;
+	border: 3px solid #ecf0f1;
+	line-height: 50px;
+	text-align: center;
+	border-radius: 100%;
+	background: rgba(0,0,0,0.5);
+	z-index: 1;
+}
+
+.tile__details:before {
+	content: '▶';
+	left: 0;
+	width: 100%;
+	font-size: 30px;
+	margin-left: 7px;
+	margin-top: -18px;
+	text-align: center;
+	z-index: 2;
+}
+
+.tile:hover .tile__details {
+	opacity: 1;
+}
+
+.tile__title {
+	position: absolute;
+	bottom: 0;
+	padding: 10px;
+}
+
+.row__inner:hover {
+	transform: translate3d(-125px, 0, 0);
+}
+
+.row__inner:hover .tile {
+	opacity: 0.3;
+}
+
+
+
+/*.row__inner:hover .tile:hover {
```

Comment:
Please remove comments from pull requests - it makes it harder to see what's important in the PR.  You can always stash the comments using github so that you can restore them for later use.